server/cluster.go

```diff
@@ -557,6 +557,28 @@ func (c *RaftCluster) checkStores() {
 	}
 }
 
+// RemoveTombStoneRecords remove the tombStone Records.
```

Comment:
s/remove/removes/g

Reply:
could you please tell me what the meaning of "s/remove/removes/g"? little confused. Thanks you!

Reply:
@rleungx

Reply:
The verbs, which use a function name as the first person, should use the third-person-singular. e.g.
```suggestion
// RemoveTombStoneRecords removes the tombStone Records.
```
They should be changed all in this PR.